### ISSUE TITLE
Jupyter books on the website are not from neurolibre organization

### ISSUE THREAD

**Issue by ltetrel** [neurolibre/neurolibre.com#25]
the submissions on https://www.neurolibre.com/ points to various location:
* https://simexp.github.io
* https://mathieuboudreau.github.io
* http://qmrlab.org
* https://brainhack101.github.io

They should all point to https://neurolibre.github.io instead to disable bypassing our submission workflow.

**Comment by ltetrel** [comment]
I temporarily disabled the `banned_specs` config to allow for these notebooks to build correctly on our binderhub.

**Comment by emdupre** [comment]
At least one has been forked already, no? https://github.com/neurolibre/vcog_hps_ad_book

The others could either be forked or we could "submit" them to test out the submission workflow.

**Comment by agahkarakuzu** [comment]
@emdupre the submit workflow is available at https://github.com/roboneurotest right now, I can add them to NeuroLibre organization for testing.

**Comment by emdupre** [comment]
If we can use this as a test of the submission workflow, I think that'd be great !

One complication is that at least `introML-book` and `vcog_hps_ad_book` are already converted into Jupyter Books, so the `build` step of our workflow wouldn't work.

@ltetrel is right that we also have the `introML` repository, which would be a more accurate test. I'm not sure if there's something equivalent for `vcog_hps_ad`. Do you know, @ltetrel ?

**Comment by ltetrel** [comment]
Yes I already converted it to jupyter book, but I could also take the raw repo https://github.com/SIMEXP/vcog_hps_ad and we could test on that.

**Comment by emdupre** [comment]
@agahkarakuzu can correct me if I'm wrong, but I think we should in fact be working with the original repositories for roboneuro :robot: to work off of !

**Comment by agahkarakuzu** [comment]
@emdupre yes, the only reason I kept script in a different repo was not to bother people with thousands of notifications :) 

I can move them here sometime tomorrow, would that be ok?
